src/modules/commander/esc_calibration.cpp

```diff
@@ -75,11 +78,97 @@ bool check_battery_disconnected(orb_advert_t *mavlink_log_pub)
 	return false;
 }
 
-int do_esc_calibration(orb_advert_t *mavlink_log_pub)
+static void set_motor_actuators(uORB::Publication<actuator_test_s> &publisher, float value, bool release_control)
+{
+	actuator_test_s actuator_test{};
+	actuator_test.timestamp = hrt_absolute_time();
+	actuator_test.value = value;
+	actuator_test.action = release_control ? actuator_test_s::ACTION_RELEASE_CONTROL : actuator_test_s::ACTION_DO_CONTROL;
+	actuator_test.timeout_ms = 0;
+
+	for (int i = 0; i < actuator_test_s::MAX_NUM_MOTORS; ++i) {
+		actuator_test.function = actuator_test_s::FUNCTION_MOTOR1 + i;
+		publisher.publish(actuator_test);
+	}
```

Comment:
If we have motor numbers higher than Queue length of `actuator_test` uORB message's topic, wouldn't the first few motors not be commanded as they get removed in the queue in FIFO manner?